webui/src/lib/components/repositoryCreateForm.jsx

```diff
@@ -15,19 +18,24 @@ export const RepositoryCreateForm = ({ config, onSubmit, onCancel, error = null,
     const repoValidityRegex = /^[a-z0-9][a-z0-9-]{2,62}$/;
 
     const [formValid, setFormValid] = useState(false);
-    const [repoValid, setRepoValid] = useState(true);
+    const [repoValid, setRepoValid] = useState(null);
+    const defaultNamespacePrefix = config.default_namespace_prefix
 
-    const [storageNamespaceValid, setStorageNamespaceValid] = useState(true);
+    const [storageNamespaceValid, setStorageNamespaceValid] = useState(defaultNamespacePrefix ? true : null);
```

Comment:
Not sure the prefix itself is necessarily "valid"; don't we still need to check?